tests/bdd/features/snapshot/list/list.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Volume Snapshot listing
+
+  Background:
+    Given a deployer cluster
+
+  # SIMPLE LIST OPERATION
+  Scenario Outline: Volume snapshot list
+    Given we have a single replica <publish_status> volume
+    And we have created a snapshot for the volume
+    Then the snapshot should be listed by snapshot id
+    Examples:
+      | publish_status |
+      | published      |
+      | unpublished    |
+
+  # FILTERED LISTING
+  Scenario Outline: Volume snapshots list
+    Given we have a single replica <publish_status> volume
+    And we have created more than one snapshot for the volume
+    Then the snapshots should be listed by <filter>
+    Examples:
+      | publish_status | filter        |
+      | published      | by_none        |
+      | published      | by_volume      |
+      | published      | by_volume_snap |
+      | unpublished    | by_none        |
+      | unpublished    | by_volume      |
+      | unpublished    | by_volume_snap |
```

Comment:
Actually, I should better merge these two scenario when I look at them again from a 1000ft.